src/Entity.php

```diff
@@ -1992,6 +1995,93 @@ public static function showInventoryOptions(Entity $entity)
         }
         echo "</td><td colspan='2'></td></tr>";
 
+        echo "</table>";
+
+        echo "<br><table class='tab_cadre_fixe'>";
+        echo "<tr>";
+        echo "<th colspan='4'>" . __('Automatically update of the elements related to the computers');
+        echo "</th><th colspan='2'>" . __('Unit management') . "</th></tr>";
+
+        echo "<tr><th>&nbsp;</th>";
+        echo "<th>" . __('Alternate username') . "</th>";
+        echo "<th>" . User::getTypeName(1) . "</th>";
+        echo "<th>" . Group::getTypeName(1) . "</th>";
+        echo "<th>" . Location::getTypeName(1) . "</th>";
+        echo "<th>" . __('Status') . "</th>";
+        echo "</tr>";
+
+        $fields = ["contact", "user", "group", "location"];
+        echo "<tr class='tab_bg_2'>";
+        echo "<td> " . __('When connecting or updating') . "</td>";
+        $values = [
+            __('Do not copy'),
+            __('Copy')
```

Comment:
Forcing keys (values) could make them more clear, and could also prevent unwanted issues during a future refactoring.
```suggestion
            0 => __('Do not copy'),
            1 => __('Copy')
```